pkg/controllers/disruption/expiration_test.go

```diff
@@ -76,6 +77,50 @@ var _ = Describe("Expiration", func() {
 		})
 		nodeClaim.StatusConditions().MarkTrue(v1beta1.Expired)
 	})
+	Context("Metrics", func() {
+		var eligibleNodesMetric string
+		var eligibleNodesLabels = map[string]string{
+			"method":             "expiration",
+			"consolidation_type": "",
+		}
+		BeforeEach(func() {
+			eligibleNodesMetric = ExpectFullyQualifiedNameFromCollector(disruption.EligibleNodesGauge)
+		})
+		It("should correctly report eligible nodes", func() {
```

Comment:
I understand making this its own separate test is nice to test individual things, but I'd rather not add more tests (when we're already having to increase the timeouts). Can you just add this eligible metric check into the existing disruption tests? If you can do this for each of the disruption tests, it'll make sure that the metric is working properly in all the different ways we're testing the codepaths too.

Reply:
I had a pretty large set of updates for the disruption suite in general in my original consolidation race condition fix PR that I'm going to incorporate into a new PR. The main change was a rework of how we handle faking the clock which significantly sped up the test suite (~5x speed improvement IIRC). I think with that coming as well we can justify a standalone test, but I could also see incorporating this elsewhere.

Reply:
Synced offline, opting for followups to solve this problem